org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/pom.xml

```diff
@@ -17,9 +17,19 @@
     <parent>
         <artifactId>org-eclipse-che-jdt-ls-extension</artifactId>
         <groupId>org.eclipse.che.ls.jdt</groupId>
-        <version>0.0.1-SNAPSHOT</version>
+        <version>0.0.2-SNAPSHOT</version>
     </parent>
     <artifactId>jdt.ls.extension.api</artifactId>
+    <version>0.0.2-SNAPSHOT</version>
```

Comment:
Isn't that inherited from the parent? At least in eclipse, it complains about "duplicated version".

Reply:
this may be redundant. I think we can drop it.

Reply:
removed.